lib/matplotlib/bezier.py

```diff
@@ -239,15 +239,114 @@ def __init__(self, control_points):
         coeff = [math.factorial(self._N - 1)
                  // (math.factorial(i) * math.factorial(self._N - 1 - i))
                  for i in range(self._N)]
-        self._px = self._cpoints.T * coeff
+        self._px = (self._cpoints.T * coeff).T
```

Comment:
Transpose in order to use "@" in `__call__`. That's the only place this was used before anyway.